scripts/mammals.js

```diff
@@ -0,0 +1,28 @@
+const mammals = [
+    {
+      name: "Tasmanian Devil",
+      group: "mammals",
+     image: "'\images\transparent-img\tasmanian-devil.png'",
```

Comment:
this url link is going to be inserted in img-htm sourcel. so have to think from the html perspective path. ex: ..\images\pic...

Reply:
hi,i made the changes please have a look thanks